packages/react-components/review/react-components.api.md

```diff
@@ -47,7 +48,7 @@ export type CallParticipant = CommunicationParticipant & {
 export const CameraButton: (props: CameraButtonProps) => JSX.Element;
 
 // @public
-export interface CameraButtonProps extends IButtonProps {
+export interface CameraButtonProps extends ControlBarButtonProps {
     localVideoViewOption?: VideoStreamOptions;
     onToggleCamera?: (options?: VideoStreamOptions) => Promise<void>;
     showLabel?: boolean;
```

Comment:
did you forget to remove this showLabel?

Reply:
Updated 👍